### ISSUE TITLE
[Feature Request]: Turn on plug when <Connect> pressed

### ISSUE THREAD

**Issue by puterboy** [jneilliii/OctoPrint-TPLinkSmartplug#307]
Would be great if printer could also turn on automatically when "Connect" is pressed -- this is the converse of and complementary to the existing functionality of connect after turning plug on.

Similarly, would be great if pressing <Run> would automatically turn on printer...
This is helpful because, I don't necessarily want to turn on the printer on "load", but I do want it to go on when I am ready to "run".

Not a major need but just would save the "click" of having to remember and then click to turn on the printer first...

BTW - great plugin!

**Comment by jneilliii** [comment]
Yeah, good idea. I personally lean on the power on with upload option since I always send the files from a slicer.
